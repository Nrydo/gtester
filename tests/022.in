8572769
479481601
852280585