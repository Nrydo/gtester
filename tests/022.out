9206563956070495